### ISSUE TITLE
Python3?

### ISSUE THREAD

**Issue by Scrut1ny** [xillwillx/skiptracer#66]
Please update to python 3 already it’s been long enough 😂

**Comment by 2thecurve** [comment]
I thought it moved to gitlabs but it doesn't work there either

**Comment by Scrut1ny** [comment]
Yeah, IDK what they're doing. All I've ever wanted to do is use this at least once without a fucking error xD
